### PR TITLE
Add Salary service

### PR DESCRIPTION
**Pull request by pehala**
- Add tooling for tests
- Transaction management sucks in tests

### CODE REVIEW

**Review by ZoltanFridrich — APPROVED** [review]
Seems to me that you have a mixture of tabs and spaces in your code, otherwise I dont see an issue.

**Comment by pehala** [comment]
> 
> 
> Seems to me that you have a mixture of tabs and spaces in your code, otherwise I dont see an issue.

Could you please point me where? It might be a configuration issue of my IDEA

File: salary/src/main/java/cz/fi/muni/pv217/employee/management/service/LeaveService.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+@RegisterRestClient(configKey="leave-api")
+@ApplicationScoped
+@Path("api/leave")
+@Produces(MediaType.APPLICATION_JSON)
+public interface LeaveService {
+
+    @GET
+    @Path("/employee/{id}")
+    List<Leave> getLeaveInPeriod(@PathParam("id") long id,
```
Comment (milosferencik) [inline salary/src/main/java/cz/fi/muni/pv217/employee/management/service/LeaveService.java:23]: I prefer Leaves instead of Leave

Comment (pehala) [reply salary/src/main/java/cz/fi/muni/pv217/employee/management/service/LeaveService.java:23]: https://www.wordhippo.com/what-is/the-plural-of/annual_leave.html It seems that annual leave is uncountable therefore plural is still Leave

**Pull request merged**